### ISSUE TITLE
needsToScan -> error Error: Protocol error (Runtime.callFunctionOn): Target closed.

### ISSUE THREAD

**Issue by realunreals** [open-wa/wa-automate-nodejs#1028]
Please check my code, screenshot and video. 

**Also after the screenshots I have added ev.on in the code and please check below**
> **EV Cannot read property 'user' of undefined test_session STARTUP**

**Code**
```
const create = require('@open-wa/wa-automate').create;
const Client = require('@open-wa/wa-automate').Client;
const decryptMedia = require('@open-wa/wa-automate').decryptMedia;
const ev = require('@open-wa/wa-automate').ev;
const smartUserAgent = require('@open-wa/wa-automate').smartUserAgent;

const mime = require('mime-types');
const fs = require('fs');
const ON_DEATH = require('death');
let globalClient;


ON_DEATH(async function(signal, err) {
	if(globalClient)await globalClient.kill();
})

async function createWebClient(){
	const weboptions = {
		sessionId:'test_session',
		useChrome: true,
		headless:false,
		throwErrorOnTosBlock:false,
		qrTimeout:40,
		authTimeout:40,
		killProcessOnBrowserClose: false,
		skipBrokenMethodsCheck: true,
		autoRefresh:true, //default to true
		qrRefreshS:15, 		
	};
	return startWeb(await create(weboptions));
}

async function startWeb(client) {
  globalClient=client;

  globalClient.onStateChanged(state=>{
	  console.log("State:",state)
    if(state==="CONFLICT" || state==="UNLAUNCHED") client.forceRefocus();
  });
}

createWebClient()
  .then(() => console.log('Finished'))
  .catch(err => console.error(err));
```

**Screesnhot**

<img width="1340" alt="Screen Shot 2020-11-04 at 15 58 36" src="https://user-images.githubusercontent.com/13606459/98115898-8fc80c00-1eb8-11eb-89b9-36880cb29b7b.png">

**Video**
First step  WA_VERSION: '2.2043.21' second step  WA_VERSION: '2.2045.13'

[TestVideo.mp4.zip](https://github.com/open-wa/wa-automate-nodejs/files/5488002/TestVideo.mp4.zip)

**Comment by smashah** [comment]
thanks @realunreals it's clear what's going on now. Patch on the way

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.0.12](https://github.com/open-wa/wa-automate-nodejs/compare/3.0.11...3.0.12) (2020-11-04)

- Release 3.0.12 [`e1c9de3`](https://github.com/open-wa/wa-automate-nodejs/commit/e1c9de3ba35dafcb23b780b92b41005d22ced0e4)
- Patch: Ensure that Store.Me.me is not undefined #1028 [`dbbd9a4`](https://github.com/open-wa/wa-automate-nodejs/commit/dbbd9a4f502de07f461d617ad817979132c2fbbf)

**Comment by smashah** [comment]
@realunreals can you try with the latest version and report back please

**Comment by realunreals** [comment]
Hello,

I have tested now. I don't receive any error but when I called client.getMe(); me is emypty
**me: { user: '____NA' }**

```
 start -> me {
  id: '1',
  pushname: 'XXXXXXXX',
  type: 'in',
  promises: {},
  status: 'XXXXXXXX@c.us',
  profilePicThumb: 'XXXXXX@c.us',
  locale: 'en',
  ref: XXXXXXXXXXX,
  refTTL: 20000,
  wid: 'XXXXXXXXXX@c.us',
  connected: true,
  protoVersion: [ 0, 17 ],
  clientToken: 'XXXXXXXXX',
  serverToken: 'XXXXXXXXXXXXXXX',
  isResponse: 'true',
  battery: 87,
  plugged: true,
  lc: 'US',
  lg: 'en',
  locales: 'en-US',
  platform: 'android',
  phone: {
    wa_version: '2.20.200.21',
    mcc: '310',
    mnc: '016',
    os_version: '5.1.1',
    device_manufacturer: 'samsung',
    device_model: 'dream2lte',
    os_build_number: 'dream2ltexx-user 5.1.1 NMF26X 500200604 release-keys'
  },
  me: { user: '____NA' }
}
```

**Comment by smashah** [comment]
@realunreals fixed in #1037
